models/Product.js

```diff
@@ -0,0 +1,54 @@
+// import important parts of sequelize library
+const { Model, DataTypes } = require('sequelize');
+// import our database connection from config.js
+const sequelize = require('../config/connection');
+
+// Initialize Product model (table) by extending off Sequelize's Model class
+class Product extends Model { }
+
+// set up fields and rules for Product model
+Product.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    product_name: {
+      type: DataTypes.STRING(120),
+      allowNull: false,
+    },
+    price: {
+      type: DataTypes.DECIMAL,
```

Comment:
Consider adding a limit to how many digits and decimal places can be given here e.g.
```
type: DataTypes.DECIMAL(6,2)
```